### PR TITLE
[translation] initial library

### PR DESCRIPTION
**Pull request by kristapratico**
This PR adds the most updated generated code and a thin convenience layer /skeleton for the client and models. 

APIView: https://apiview.dev/Assemblies/Review/019355482c6245a5a404eef64740da1e

### CODE REVIEW

File: sdk/documenttranslation/azure-ai-documenttranslation/setup.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    license='MIT License',
+    # ensure that the development status reflects the status of your package
+    classifiers=[
+        "Development Status :: 4 - Beta",
+
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
```
Comment (kristapratico) [inline sdk/documenttranslation/azure-ai-documenttranslation/setup.py:44]: drop 3.5 support

File: sdk/documenttranslation/azure-ai-documenttranslation/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Release History
+
+## 1.0.0b1 (Unreleased)
```
Comment (maririos) [inline sdk/documenttranslation/azure-ai-documenttranslation/CHANGELOG.md:3]: oh hello!

File: sdk/documenttranslation/azure-ai-documenttranslation/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Release History
```
Comment (maririos) [inline sdk/documenttranslation/azure-ai-documenttranslation/CHANGELOG.md:1]: does the folder need to match the namespace? just wondering if this should be in `translation`  or something else.
For now it is fine

File: sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_client.py [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+            credential=credential,  # type: ignore
+            api_version=self._api_version,
+            sdk_moniker=USER_AGENT,
+            authentication_policy=authentication_policy,
+            polling_interval=5,  # TODO what is appropriate polling interval
+            **kwargs
+        )
+
+    @distributed_trace
+    def create_translation_job(self, batch, **kwargs):
```
Comment (maririos) [inline sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_client.py:48]: pretty skeleton :P

File: sdk/documenttranslation/azure-ai-documenttranslation/setup.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+setup(
+    name=PACKAGE_NAME,
+    version=version,
+    description='Microsoft Azure {} Client Library for Python'.format(PACKAGE_PPRINT_NAME),
+
+    # ensure that these are updated to reflect the package owners' information
+    long_description=long_description,
+    url='https://github.com/Azure/azure-sdk-for-python',
+    author='Microsoft Corporation',
+    author_email='azuresdkengsysadmins@microsoft.com',
```
Comment (maririos) [inline sdk/documenttranslation/azure-ai-documenttranslation/setup.py:33]: didn't know this was a thing

File: sdk/documenttranslation/tests.yml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+trigger: none
+
+jobs:
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
```
Comment (maririos) [inline sdk/documenttranslation/tests.yml:4]: will this add a pipeline for nightly test? will the samples start running?
I just wonder how the cleanup of the target container will work

**Review by maririos — APPROVED** [review]
Only clarifying comment is pipeline and running samples

File: sdk/documenttranslation/azure-ai-documenttranslation/CHANGELOG.md (hunk 2 above)
Comment (kristapratico) [reply sdk/documenttranslation/azure-ai-documenttranslation/CHANGELOG.md:3]: 👋

File: sdk/documenttranslation/azure-ai-documenttranslation/CHANGELOG.md (hunk 3 above)
Comment (kristapratico) [reply sdk/documenttranslation/azure-ai-documenttranslation/CHANGELOG.md:1]: Still part of the larger namespace issue. Might move around later.

File: sdk/documenttranslation/azure-ai-documenttranslation/setup.py (hunk 5 above)
Comment (kristapratico) [reply sdk/documenttranslation/azure-ai-documenttranslation/setup.py:33]: This was just copied straight from our template package

File: sdk/documenttranslation/tests.yml (hunk 6 above)
Comment (kristapratico) [reply sdk/documenttranslation/tests.yml:4]: No, only the ci pipeline will be created for recorded tests. We still need to figure out how we'll do creation/tear down of live resources. I'm going to look into that today a bit.

**Pull request merged**